onboarding/managers.md

```diff
@@ -5,21 +5,32 @@ description: Things for managers to remember during the early days of onboarding
 
 ## Onboarding For Managers
 
+The manager's responsibility during on-boarding is to select the people and prepare the tools so that the 2 months of on-boarding will run as smoothly as possible.
+
 ### Before Day 1
 
 - [ ] Choose a peer mentor for the new hire
 - [ ] Choose a spirit guide for the new hire (consult with People Ops)
 - [ ] Create [an onboarding github issue](https://github.com/artsy/README/issues/new?template=engineering-onboarding.md) in this repo based on the onboarding template. Also invite the team to say hi, as this is a place that the new hire should always be able to find easily in the future.
 - [ ] Set first-week schedule and let the new hire know their first day schedule on their private email account, as there will be a period where their machine is still being setup and they won’t yet have access to their calendar.
 - [ ] Invite new hire to all engineering-wide meetings
+- [ ] Invite new hire to a 1:1 on day 1
 - [ ] Determine [sprint rotation plan](/onboarding/sprint-rotation.md)
+- [ ] Remind tech leads/PMs that are part of the sprint rotation to prepare resources and invite new hire to meetings
+- [ ] Familiarize yourself with people resources across the company that will be important to your mentee:
+  - Point people in Workplace & People Ops
+  - Their manager
+  - Their team lead
```

Comment:
I'm not sure what this means in this context (they don't have a "team" yet necessarily)